qiskit/synthesis/evolution/suzuki_trotter.py

```diff
@@ -104,6 +105,9 @@ def __init__(
                 built.
             wrap: Whether to wrap the atomic evolutions into custom gate objects. This only takes
                 effect when ``atomic_evolution is None``.
+            preserve_order: Whether to allow reordering the terms of the operator to
+                potentially yield a shallower evolution circuit. Not relevant
+                when synthesizing operator with a single term. Defaults to ``True``.
```

Comment:
Now I like the docstring 😄 

Actually, the way this is worded, it feels the other way around, i.e. `preserve_order` restricts the order to the original order.

Reply:
Fixed in 56e619c